src/core/Microsoft.PowerToys.Settings.UI.Lib/PowerLauncherProperties.cs

```diff
@@ -46,7 +46,7 @@ public class PowerLauncherProperties
 
         public PowerLauncherProperties()
         {
-            OpenPowerLauncher = new HotkeySettings();
+            OpenPowerLauncher = new HotkeySettings(false, false, true, false, 68);
```

Comment:
The last argument should be VK_SPACE (for C#, `Windows.System.VirtualKey.Space`) which is 32.

Reply:
Absolutely! I don't know why I put 68 there

Reply:
how does the logic currently do the default value for launcher?  I feel like this should be a shared default constant / static readonly

Reply:
currently it has some defaults in Settings.cs and that would be overwritten by loaded setting - however if you created this properties class and save it, it would nullify previous settings. 

We should probably handle settings in one class only (this one). I guess the other class (Settings.cs) exists for historical reasons and should not have any defaults in there - read them from this class instead?

Reply:
@crutkas I think we should create a separate issue for this to reduce the code duplication.